bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/graphics/Path.java

```diff
@@ -695,4 +699,20 @@ public String toString() {
 	return "Path {" + handle + "}";
 }
 
+/**
+ * Gets the handle for the path at the specified zoom level
+ *
+ * @since 3.126
+ */
+public static long win32_getHandle(Path path, int zoomLevel) {
+	if(path.handleMap.get(zoomLevel) == null) {
```

Comment:
Can you add a whitespace between if and (?

Reply:
Added